torch_geometric/loader/utils.py

```diff
@@ -206,12 +207,24 @@ def filter_feature_store(
         data[str_to_edge_type(key)].edge_index = edge_index
 
     # Filter node storage:
-    for attr in feature_store.get_all_tensor_attrs():
-        if attr.group_name in node_dict:
-            # If we have sampled nodes from this group, index into the
-            # feature store for these nodes' features:
-            attr.index = node_dict[attr.group_name]
-            tensor = feature_store.get_tensor(attr)
-            data[attr.group_name][attr.attr_name] = tensor
-
+    attrs = feature_store.get_all_tensor_attrs()
```

Comment:
or I'd suggest that we do
```
tensors = feature_store.multi_get_tensor(attrs)
for i, attr in enumerate(attrs):
  data[attr.group_name][attr.attr_name] = tensors[i]
```
Because `multi_get_tensor` is synchronized, doing it this way will parallel the get in the for loop `for group_name in node_dict` in C++ backend. (If not, we can change that. Feature store owns the optimization)